mautic/mautic
# Disable trackable urls

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/4220
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Just added configuration option to disable trackable urls for emails (read more https://github.com/mautic/mautic/issues/4220). 

#### Steps to test this PR:
1.  Apply and clear cache
2.  Go to configurations > Emails and enable options "Disable trackable urls"
3. Send email throught form and see if urls don't covert to /r/hash format.
